example/pythonlib/module/1-common-config/build.mill

```diff
@@ -0,0 +1,77 @@
+package build
+import mill._, pythonlib._
+
+// TODO: When we use `package` instead of foo then it fails for `show` command in CI test Why?
```

Comment:
If you use `package`, then you'd need to update the command `mill foo.XXX` to `mill XXX`. Does that still fail?

Reply:
Yaa i was doing exactly the same thing but its was not working

Reply:
>   /**
>   * Command-line options to pass to the Python Interpreter defined by the user.
>   */
any more suggestion why it's not working